torchtitan/config_manager.py

```diff
@@ -369,6 +370,27 @@ def __init__(self):
             help="Python garbage control scheduling interval, in steps",
         )
 
+        # validation configs
+        self.parser.add_argument(
+            "--validation.batch_size", type=int, default=0
```

Comment:
The default for validation batch size should be the training batch size. Here let's have the default as None, and add a help discription that explains it defaults to the training batch size when not passed.